src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -199,13 +223,42 @@ public async Task<IActionResult> GetSeriesMetadataAsync([FromHeader(Name = IfNon
         return CreateResult(response);
     }
 
-    private static IActionResult CreateResult(RetrieveMetadataResponse response)
+    [Produces(KnownContentTypes.ImageJpeg)]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NoContent)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotAcceptable)]
+    [HttpGet]
+    [VersionedPartitionRoute(KnownRoutes.FrameRenderedRoute)]
+    [VersionedRoute(KnownRoutes.FrameRenderedRoute)]
+    [AuditEventType(AuditEventSubType.RetrieveRendered)]
+    public async Task<IActionResult> GetRenderedFrameAsync(
+        string studyInstanceUid,
+        string seriesInstanceUid,
+        string sopInstanceUid,
+        int frames)
```

Comment:
frame? Why is it plural?

Reply:
Was intending to match dicom standard with `frames` and then specify in conformance statement that we only support one value but updated route to be only `frame` instead.